altislife.sql

```diff
@@ -12,12 +12,14 @@ SET time_zone = "+00:00";
 /*!40101 SET NAMES utf8mb4 */;
 
 --
--- Database: `altislife`
+-- Creates database `altislife` unless it already exists and uses `altislife`
 -- Default Schema
+-- Creates default user `arma3` with password `changeme` unless it already exists
 --
 CREATE DATABASE IF NOT EXISTS `altislife` DEFAULT CHARACTER SET utf8mb4;
 USE `altislife`;
 
+CREATE USER IF NOT EXISTS `arma3`@`localhost` IDENTIFIED BY 'changeme';
```

Comment:
It would be good for security to append [`PASSWORD EXPIRE`](https://dev.mysql.com/doc/refman/5.7/en/alter-user.html#alter-user-password-management) so the user is forced to change the user password. Unfortunately, [MariaDB](https://mariadb.org/) does not currently (version 10.2.8) support this value. Source: [MDEV-7597 - Expiration of user passwords](https://jira.mariadb.org/browse/MDEV-7597)

Reply:
Could be, but some users will find this kinda useless, and then re-set it up to the original they wrote in the file.